src/groupeddataframe/splitapplycombine.jl

```diff
@@ -33,7 +33,8 @@ and combines the result into a data frame).
 which can also be used to get the values of the grouping columns for each group.
 `Tuples` and `NamedTuple`s containing the values of the grouping columns (in the
 same order as the `cols` argument) are also accepted as indices, but this will
-be slower than using the equivalent `GroupKey`.
+be slower than using the equivalent `GroupKey`. Finally, a `Dict` can be used to index
+into a grouped data frame where the keys are column names of the data frame in any order. 
```

Comment:
```suggestion
be slower than using the equivalent `GroupKey`. Finally, a `Dict` with
column names as keys can be used (order does not matter in this case).
Note that indexing with `GroupKey` is faster than other methods.
```
(And remove the mention of slower above for tuples and named tuples.)